### ISSUE TITLE
storyboard: error: IB Designables: Failed to update auto layout status

### ISSUE THREAD

**Issue by tbechtum** [cbpowell/MarqueeLabel#140]
Hi to everbody,

did pod update "Installing MarqueeLabel 2.7.5 (was 2.7.2)" what this is resulting in the latest Xcode Version 7.3 (7D175) to storyboard: error: IB Designables: Failed to update auto layout status: Failed to load designables from path (null).

Tried to resolve it following stackoverflow.com/questions/26799054/failed-to-render-instance-of-ib-designables, but these steps did not resolve this issue.

Thanks for your help,
Thomas

**Comment by tbechtum** [comment]
Tried also versions 2.7.3 and 2.7.4 and they are creating the same issue. Do I have to recreate all Marquee labels in the storyboard?

**Comment by songhailiang** [comment]
Designables updating

**Comment by songhailiang** [comment]
same error for me!

**Comment by cbpowell** [comment]
Are you guys using the Swift code? I've noticed some errors with IBDesignable as well, but so far haven't put too much effort into solving it. The Objective-C version seems fine, oddly.

**Comment by cbpowell** [comment]
Also, if you're using this on tvOS with Carthage, [release 2.7.4](https://github.com/cbpowell/MarqueeLabel/releases/tag/2.7.4) changed the tvOS framework name - arguably that should have been a 2.8 major version bump. That could maybe (just guessing) cause your issue if you have a specific module set in the Identity inspector pane in Interface Builder:
<img width="257" alt="screen shot 2016-05-30 at 12 03 01 pm" src="https://cloud.githubusercontent.com/assets/593953/15656662/827be424-265e-11e6-8021-2386fa670988.png">

On a similar note, I was able to clear some of the warnings I was getting from using IBDesignable in the demo by removing an outdated runtime attribute for "MarqueeType". Swift uses the `type` property, which as an enum can't currently be set in IB. Removing the selected line (with the "-" button) removed quite a few warnings in the demo:
<img width="262" alt="screen shot 2016-05-30 at 12 05 46 pm" src="https://cloud.githubusercontent.com/assets/593953/15656738/1a475888-265f-11e6-871d-bf386485bb23.png">

**Comment by tbechtum** [comment]
Hi Charles,

thanks a lot for your reply. First I introduced the MarqueeLabel in a Objective-C project late 2014 with Cocoapods integration. Now I am working in a swift project and I noticed later on that there is the MarqueeLabel/Swift solution. When I migrated from MarqueeLabel to MarqueeLabel/Swift I was confronting the described issues.
Then I updated also the version of MarqueeLabel in the Objective-C project and there were the same issues.

Now I found this solution:
I had to completely remove MarqueeLabel from the Storyboard and the related view controller code, make a clean build. Afterwards re-introduce MarqueeLabel from scratch to your project and the problem will be resolved.

Cheers,
Thomas

On 30 May 2016, at 21:09, Charles Powell <notifications@github.com<mailto:notifications@github.com>> wrote:


Also, if you're using this on tvOS with Carthage, release 2.7.4<https://github.com/cbpowell/MarqueeLabel/releases/tag/2.7.4> changed the tvOS framework name - arguably that should have been a 2.8 major version bump. That could maybe (just guessing) cause your issue if you have a specific module set in the Identity inspector pane in Interface Builder:
[screen shot 2016-05-30 at 12 03 01 pm]<https://cloud.githubusercontent.com/assets/593953/15656662/827be424-265e-11e6-8021-2386fa670988.png>

On a similar note, I was able to clear some of the warnings I was getting from using IBDesignable in the demo by removing an outdated runtime attribute for "MarqueeType". Swift uses the type property, which as an enum can't currently be set in IB. Removing the selected line (with the "-" button) removed quite a few warnings in the demo:
[screen shot 2016-05-30 at 12 05 46 pm]<https://cloud.githubusercontent.com/assets/593953/15656738/1a475888-265f-11e6-871d-bf386485bb23.png>
